fix: Disabled pinch to zoom on mobile devices#9155

Gohan-18 opened the pull request:
## What does this PR do?
Disables pinch to zoom in mobile devices

<!-- Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change. -->

Fixes #9155  (issue)


<!-- Please provide a loom video for visual changes to speed up reviews
 Loom Video: https://www.loom.com/
-->

**Environment**: Staging(main branch) / Production

## Type of change

<!-- Please delete bullets that are not relevant. -->

#- Bug fix (non-breaking change which fixes an issue)
#- Chore (refactoring code, technical debt, workflow improvements)
- New feature (non-breaking change which adds functionality)
#- Breaking change (fix or feature that would cause existing functionality to not work as expected)
#- This change requires a documentation update

## How should this be tested?

<!-- Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration -->

- [x] Test A
- [x] Test B

## Checklist

Gohan-18 committed:
Disabled pinch to zoom on mobile devices#9155

sean-brydon commented:
@PeerRich  what is the reason for this? Its generally recommended to avoid disabling pinch to zoom as its not the best thing to do for a11y.

PeerRich commented:
@sean-brydon zooming on mobile apps looks weird and breaks the pwa experience. i dont mind keeping it, just looks weird. visit the mobile page and pinch to zoom, looks odd

PeerRich committed:
Merge branch 'main' into Gohan_18/pinchToZoonFix

PeerRich reviewed (changes requested):
@Gohan-18 isnt 

`<meta name="viewport" 
      content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0">
`

better?

Gohan-18 commented:
> 

Yeah... That's good too. Let me just modify the code. Thanks.

Gohan-18 commented:
Hey @PeerRich , one small help. Couldn't actually locate the file where to add the tag that works throughout app. Tried in both the theme.config.js and index.html, but doesn't seems to work. Can you please just help in locating the file. Thanks.

PeerRich commented:
`<meta>` should be in  _document.tsx line 51

Gohan-18 committed:
Modified the better fix to the feature

Gohan-18 committed:
Merge branch 'Gohan_18/pinchToZoonFix' of https://github.com/Gohan-18/cal.com into Gohan_18/pinchToZoonFix

Gohan-18 commented:
> `<meta>` should be in _document.tsx line 51

Just updated. Thanks again for helping out

> `<meta>` should be in _document.tsx line 51

Just updated. Thanks again for being so patience and helpful.🙏

PeerRich commented on apps/web/pages/_document.tsx:
```diff
@@ -46,8 +46,8 @@ class MyDocument extends Document<Props> {
-          <meta name="theme-color" media="(prefers-color-scheme: light)" content="#f9fafb" />
-          <meta name="theme-color" media="(prefers-color-scheme: dark)" content="#1C1C1C" />
+          <meta name="theme-color" content="var(--cal-bg)" />
+          <meta name="viewport"  content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0"></meta>
```
```suggestion
          <meta name="viewport"  content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0"/>
```

PeerRich committed:
Update apps/web/pages/_document.tsx

PeerRich committed:
Update apps/web/pages/_document.tsx

PeerRich commented on apps/web/styles/globals.css:
```diff
@@ -287,6 +287,7 @@ select:focus {
 
 /* hide chat bubble on mobile */
 @media only screen and (max-width: 768px) {
+
```
```suggestion
```

PeerRich committed:
Update apps/web/styles/globals.css

PeerRich committed:
Update apps/web/pages/_document.tsx

PeerRich committed:
Merge branch 'main' into Gohan_18/pinchToZoonFix

Gohan-18 committed:
Merge branch 'main' into Gohan_18/pinchToZoonFix

PeerRich committed:
Merge branch 'main' into Gohan_18/pinchToZoonFix

PeerRich commented on apps/web/pages/_document.tsx:
```diff
@@ -48,6 +48,7 @@ class MyDocument extends Document<Props> {
           <meta name="msapplication-TileColor" content="#ff0000" />
           <meta name="theme-color" media="(prefers-color-scheme: light)" content="#f9fafb" />
           <meta name="theme-color" media="(prefers-color-scheme: dark)" content="#1C1C1C" />
+          <meta name="viewport"  content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" />
```
```suggestion
          <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" />
```

PeerRich committed:
Update apps/web/pages/_document.tsx

keithwillcode committed:
Merge branch 'main' into Gohan_18/pinchToZoonFix

PeerRich commented on apps/web/pages/_document.tsx:
```diff
@@ -48,6 +48,7 @@ class MyDocument extends Document<Props> {
           <meta name="msapplication-TileColor" content="#ff0000" />
           <meta name="theme-color" media="(prefers-color-scheme: light)" content="#f9fafb" />
           <meta name="theme-color" media="(prefers-color-scheme: dark)" content="#1C1C1C" />
+          <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" />
```
```suggestion
          <meta
            name="viewport"
            content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0"
          />
```

PeerRich committed:
Update apps/web/pages/_document.tsx
